src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -16,11 +18,12 @@ import type { LocalFrame } from "./types";
 
 type FrameTitleProps = {
   frame: Frame,
-  options: Object
+  options: Object,
+  l10n: Object
 };
 
-function FrameTitle({ frame, options = {} }: FrameTitleProps) {
-  const displayName = formatDisplayName(frame, options);
+function FrameTitle({ frame, options = {}, l10n }: FrameTitleProps) {
+  const displayName = formatDisplayName(frame, options, l10n);
```

Comment:
I'd do something like this

```suggestion
  const displayName = formatDisplayName(frame, options) || l10n.getStr("anonymousFunction");
```

Reply:
agreed, i guess this is okay...

Reply:
not ideal for sure. that's why I did https://github.com/devtools-html/debugger.html/pull/7203 first, seemed a bit more easy to use.